components/tableNodes/NodeTable.component.tsx

```diff
@@ -55,6 +55,24 @@ export function NodeTable({ show }: NodeTableProps): JSX.Element {
     show(true, "No channels open in this node");
     return <></>;
   }
+
+  function convert(param: any): number {
+    if (param instanceof Number) {
+      return param as number;
+    }
+    return parseInt(param);
+  }
+  function check_amount_msat(param: any): string {
```

Comment:
we can put some space between the function? and take care of the js code a little bit in some of the next Prs
```suggestion
  }
  
  function check_amount_msat(param: any): string {
```